flowclient/flowclient/client.py

```diff
@@ -251,14 +338,16 @@ def get_result_by_query_id(connection: Connection, query_id: str) -> pd.DataFram
         query_ready, reply = query_is_ready(connection, query_id)  # Poll the server
 
     logger.info(f"Getting {connection.url}/api/{connection.api_version}/get/{query_id}")
-    response = connection.session.get(
-        f"{connection.url}{reply.headers['Location']}", allow_redirects=False
-    )
+    result_location = reply.headers[
+        "Location"
+    ]  # Need to strip off the /api/<api_version>/
+    result_location = "/".join(result_location.split("/")[3:])
```

Comment:
This reads a little cryptic. How about a more explicit substitution? (need to also import the `re` module)
```suggestion
result_location = re.sub("^/api/[0-9]+/", "", result_location)  # strip off the /api/<api_version>/
```
```